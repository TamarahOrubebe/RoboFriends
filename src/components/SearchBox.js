import React from 'react';


const SearchBox = ({onSearch}) => {

    return (
        <div >

            <input onChange={onSearch} className="pa3 ba b--green bg-lightest-blue" aria-label="searcbox" type="search" name="searchbox" />
        </div>
    )
}


export default SearchBox;