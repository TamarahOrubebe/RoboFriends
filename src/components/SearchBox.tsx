import React from 'react';


interface IProps{
    onSearch: VoidFunction
}

const SearchBox = ({onSearch}: IProps) => {

    return (
        <div >

            <input onChange={onSearch} className="pa3 ba b--green bg-lightest-blue" aria-label="searcbox" type="search" name="searchbox" />
        </div>
    )
}


export default SearchBox;