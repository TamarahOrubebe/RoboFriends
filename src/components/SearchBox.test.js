import React from "react";
import { shallow } from "enzyme";
import SearchBox from "./SearchBox";

it("renders a SearchBox component", () => {
	const mockRobots = [
		{
			id: 1,
			username: "Tamarah",
			name: "Paul Orubebe",
			email: "paulorubebe@gmail.com",
		},
	];

	expect(shallow(<SearchBox />)).toMatchSnapshot();
});
