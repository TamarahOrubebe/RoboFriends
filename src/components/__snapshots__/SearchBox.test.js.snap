// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a SearchBox component 1`] = `
<div>
  <input
    aria-label="searcbox"
    className="pa3 ba b--green bg-lightest-blue"
    name="searchbox"
    type="search"
  />
</div>
`;
